cmd/server/main.go

```diff
@@ -663,8 +664,10 @@ func (p *Proxy) runHealthServer(o *ProxyRunOptions, server *server.ProxyServer)
 		if ready {
 			w.WriteHeader(200)
 			fmt.Fprintf(w, "ok")
+			metrics.Metrics.SuccessInc()
```

Comment:
This metric is incremented when someone sends a GET request to the the `/ready` path. Can you elaborate on the use case of this metric?

Reply:
I probably put the metrics in the wrong place. What I want is to have a metrics showing the healthiness of konnectivity network server. I checked the liveness check, and it only returns ok without checking if the agent is running.